backend/private-graph/graph/schema.resolvers.go

```diff
@@ -3063,13 +3063,14 @@ func (r *mutationResolver) DeleteLogAlert(ctx context.Context, projectID int, id
 	}
 
 	alert := &model.LogAlert{}
-	if err := r.DB.Model(&model.LogAlert{Model: model.Model{ID: id}}).
```

Comment:
wonder if it's possible to set up a lint rule for non-zero-value structs passed to `.Model()` as that seems like always misguided